casper/src/main/scala/coop/rchain/casper/util/comm/CommUtil.scala

```diff
@@ -52,41 +56,127 @@ object CommUtil {
       _     <- TransportLayer[F].broadcast(peers, msg)
     } yield ()
 
+  def requestApprovedBlock[
+      F[_]: Monad: Capture: MultiParentCasperConstructor: Log: Time: Metrics: TransportLayer: NodeDiscovery: ErrorHandler: PacketHandler]
+    : F[Unit] = {
+    val request = ApprovedBlockRequest("PleaseSendMeAnApprovedBlock").toByteString
+
+    def askPeers(peers: List[PeerNode], local: PeerNode): F[Unit] = peers match {
+      case peer :: rest =>
+        for {
+          _    <- Log[F].info(s"CASPER: Sending request for ApprovedBlock to $peer")
```

Comment:
this is not a blocker but just so you know, each log has now a `LogSource` where you can define a common prefix for all your logs in given file